cardstack/src/hooks/prepaid-card/useBuyPrepaidCard.ts

```
@@ -389,7 +413,8 @@ export default function useBuyPrepaidCard() {
       wyreOrderIdData = await onPurchase({
         value: amount.toString(),
         depositAddress: custodialWalletData?.attributes['deposit-address'],
-        sourceCurrency: nativeCurrency,
+        sourceCurrency: nativeCurrency || card?.['source-currency'],
+        destCurrency: card?.['dest-currency'],
```

Should it default to DAI ?

---

for now that is ok. but when/if we are able to buy prepaid cards in the future backed by different tokens, it should just throw an error